application.py

```diff
@@ -66,10 +68,46 @@ def hello():
     sys.stdout.flush()
 
     try:
-        return("Notifications: {} Services: {}".format(len(notifications), len(services)))
+        response = {
+            'notifications': len(notifications),
+            'services': len(services)
+        }
+        return(response)
     except:
-        return("Had an error connecting to database")
+        resp_error = {
+            'error':"Had an error connecting to database"
+        }
+        return(resp_error)
 
 
+@application.route('/live-services')
```

Comment:
Just occurred to me, we should start a "Routes" section in the readme with a line for each of our routes.